registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/RegistrationStatusServiceImpl.java

```diff
@@ -161,6 +162,7 @@ public void addRegistrationStatus(InternalRegistrationStatusDto registrationStat
 			String transactionId = generateId();
 			registrationStatusDto.setLatestRegistrationTransactionId(transactionId);
 			registrationStatusDto.setCreateDateTime(LocalDateTime.now(ZoneId.of("UTC")));
+			registrationStatusDto.setSource("REGISTRATION_CLIENT");
```

Comment:
This should not be hard coded in code. For time being we can read from a configuration, later we will implement actual code